ui/server/src/main/scala/pl/touk/nussknacker/ui/process/deployment/ManagementActor.scala

```diff
@@ -115,8 +107,34 @@ class ManagementActor(managers: ProcessingTypeDataProvider[ProcessManager],
       }
     case DeploymentStatus =>
       reply(Future.successful(DeploymentStatusResponse(beingDeployed)))
+
+    case CustomAction(actionReq, id, user) =>
```

Comment:
We assume here there are not limitations regarding concurrency? Maybe at least a comment on that?

Reply:
https://github.com/TouK/nussknacker/pull/1367/commits/153ad134e0f6ca0858c28c15af314f83388f2b8b